vmslave-config/pipeline/all

```diff
@@ -0,0 +1,14 @@
+# If not set, default: use roles/replace-apt-source/files/sources.list
```

Comment:
.in means a file to be modified?
sources.list is a sources.list can be directly used. 
So I just use sources.list.